packages/snowpack/src/commands/dev.ts

```diff
@@ -431,7 +431,13 @@ export async function command(commandOptions: CommandOptions) {
         .catch(() => null /* ignore */);
     }
 
-    let requestedFileExt = path.parse(reqPath).ext.toLowerCase();
+    let {baseExt: requestedFileExt} = getExt(reqPath.replace(/\.map$/i, ''));
```

Comment:
somewhere above we do handling for `if (reqPath.endsWith('.proxy.js')) {`, this feels related and could be moved up to that section.

Reply:
Moved it exactly there!